client/galaxy/scripts/mvc/library/library-foldertoolbar-view.js

```diff
@@ -844,11 +857,13 @@ var FolderToolbarView = Backbone.View.extend({
   },
 
   /**
-   * Take the array of paths and createa request for each of them
-   * calling them in chain. Update the progress bar in between each.
+   * Take the array of paths and create a request for each of them
+   * calling them in series. Update the progress bar in between each.
    * @param  {array} paths                    paths relative to Galaxy root folder
    * @param  {boolean} preserve_dirs          indicates whether to preserve folder structure
    * @param  {boolean} link_data              copy files to Galaxy or link instead
+   * @param  {boolean} to_posix_lines              copy files to Galaxy or link instead
```

Comment:
Comment should be updated here probably (copied from above when editing, looks like)

Reply:
resolved, thanks!